packages/mui-styled-engine-sc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/styled-engine-sc",
-  "version": "6.0.0-alpha.18",
+  "version": "6.0.0-alpha.19",
```

Comment:
Why is this using the `6.0.0` prefix? 😅

Reply:
That's a good point 😱. I will revert this one.

Reply:
> But don't we need this to be released for SC users to be able to use CSS layers?

SC users cannot use CSS layers. We currently support Emotion only (this is the limitation of the APIs the library expose).